src/post-message.ts

```diff
@@ -610,11 +610,14 @@ export const postMessageToNativeApp = <T extends keyof ResponsesFromNativeApp>(
 };
 
 /**
- * Initiates WebApp postMessage function, which will be called by native apps
+ * Initiates postMessage function, which will be called by native apps
  */
 if (typeof window !== 'undefined') {
-    window[BRIDGE] = window[BRIDGE] || {
```

Comment:
This will work in these scenarios:
- both instances of the bridge are using this or newer version
- in case one of the instances is using an older version:
  - if the old version is loaded first, it will work
  - if the old version is loaded after the new one, only the new one will work